### PR TITLE
Resolves #21 - Binutils error

### LINKED ISSUE #21: AttributeError: 'dict' object has no attribute '__NUMPY_SETUP__'
```
  File "setup.py", line 39, in finalize_options
    __builtins__.__NUMPY_SETUP__ = False
AttributeError: 'dict' object has no attribute '__NUMPY_SETUP__'
```
[Builtins](https://github.com/pyvista/pyacvd/blob/73cedc15fcf5151e05647b6741a250de11b985d9/setup.py#L39) is not innately supported : Referring to this [Example](https://www.programcreek.com/python/example/67000/builtins.__NUMPY_SETUP__)

### PR DESCRIPTION
**Pull request by icanhasmath**
This change adds a fallback for a binutils error.  Also adds a cython dependency for install.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -78,7 +85,8 @@ def read(*paths):
     ],
 
     python_requires='>=3.6.*',
-    install_requires=['pyvista>=0.32.0',
+    install_requires=['cython',
```
Comment (akaszynski) [inline setup.py:88]: Really, we should be using `cython` within a `pyproject.toml` build requires section.

Comment (icanhasmath) [reply setup.py:88]: Agreed, I can drop this commit from the PR if you're updating to a `pyproject.toml`

Comment (icanhasmath) [reply setup.py]: Dropped.

**Comment by akaszynski** [comment]
Closing in favor of #23.

**Pull request closed without merging**